README.md

```diff
@@ -1,68 +1,32 @@
-# MinIO Operator Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
+# MinIO Operator [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
 
-MinIO is a high performance distributed object storage server, designed for large-scale private cloud infrastructure. MinIO is designed in a cloud-native manner to scale sustainably in multi-tenant environments. Orchestration platforms like Kubernetes provide perfect launchpad for MinIO to scale.
+MinIO Operator brings native support for [MinIO](https://github.com/minio/minio), [Console](https://github.com/minio/console), and [KES](https://github.com/minio/kes) to Kubernetes.
 
-MinIO-Operator brings native MinIO, [Console](https://github.com/minio/console), and [KES](https://github.com/minio/kes) support to Kubernetes. MinIO-Operator currently supports following features:
+## Prerequisites
 
-| Feature                 | Reference Document |
-|-------------------------|--------------------|
-| Create and delete highly available distributed MinIO clusters  | [Create a MinIO Instance](https://github.com/minio/operator#create-a-minio-instance). |
-| TLS Configuration  | [TLS for MinIO Instance](https://github.com/minio/operator/blob/master/docs/tls.md). |
-| Expand an existing MinIO cluster | [Expand a MinIO Cluster](https://github.com/minio/operator/blob/master/docs/adding-zones.md). |
-| Use a custom template for hostname discovery | [Custom Hostname Discovery](https://github.com/minio/operator/blob/master/docs/custom-name-templates.md). |
-| Use PodSecurityPolicy for MinIO Pods | [Apply PodSecurityPolicy](https://github.com/minio/operator/blob/master/docs/pod-security-policy.md). |
-| Deploy Console with MinIO cluster  | [Deploy MinIO Instance with Console](https://github.com/minio/operator/blob/master/docs/console.md). |
-| Deploy KES with MinIO cluster  | [Deploy MinIO Instance with KES](https://github.com/minio/operator/blob/master/docs/kes.md). |
+- Kubernetes >= v1.17.0.
+- Create PVs using [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md).
```

Comment:
Should we provide some guidance here? Number of recommended drives, for example. Or something like "Create One PV for each PVC in the cluster."

That also kind of requires someone to manually do the math on how many volumes they'll end up with based on `servers` * `volumes`, if I've got the right idea.

Reply:
I think this makes it sound like `Direct CSI` is required to use the Operator, can we make it more like a recommendation? since PVs are part of kubernetes, they can be created any different forms, so we may want to suggest that it's recommended to have something that can provision drives to `PersistentVolumeClaims` created by the operator